app/models/user.rb

```diff
@@ -73,6 +74,17 @@ class User < ApplicationRecord
     :authenticatable_salt,
     :invited_to_sign_up?, to: :email_authentication, allow_nil: true
 
+  delegate :organizations,
+    :facilities,
+    :facility_groups, to: :accesses, prefix: :accessible
+
+  delegate :can_manage_facilities?,
+    :can_manage_facility_groups?,
+    :can_manage_organizations?,
+    :can_view_identifiable_info?,
```

Comment:
Is this for PII?  If so lets be explicit about it, its a critical piece of the auth levels:

```
admin.can_view_PII?
```

Or

```
admin.can_view_patient_information?
```

Reply:
`admin.can_view_patient_information?` is a lot better 👍